plugin/forward/README.md

```diff
@@ -83,6 +84,8 @@ forward FROM TO... {
   * `round_robin` is a policy that selects hosts based on round robin ordering.
   * `sequential` is a policy that selects hosts based on sequential ordering.
 * `health_check`, use a different **DURATION** for health checking, the default duration is 0.5s.
+* `max_concurrent` **MAX** will limit the number of concurrent queries to **MAX**.  Any new query that would
```

Comment:
how should one determine this value, what's a good one for **MAX**?

Reply:
We should probably add words on why we're not nacking health if this happens (i.e. to avoid cascading failure)

Reply:
I put in notes for these.  For picking a value, I describe a lower bound.  

I didn't tackle describing an upper bound yet.  For an upper bound (to bind memory usage), it's a matter of how much memory an average query takes up inflight in CoreDNS - which is variable (e.g. encryption).  I can try to estimate this experimentally.  I don't know how globally applicable or accurate that would be.